regression-test/data/nereids_function_p0/array/test_array_array_bigint.csv

```diff
@@ -0,0 +1,11 @@
+[[]]
+[[null], [null], [null], [null], [null], [null], [null], [null], [null], [null]]
+NULL
+[[0], [0], [0], [0], [0], [0], [0], [0], [0], [0]]
+[[-9223372036854775808, 9223372036854775807], [-9223372036854775808, 9223372036854775807], [-9223372036854775808, 9223372036854775807], [-9223372036854775808, 9223372036854775807], [-9223372036854775808, 9223372036854775807], [-9223372036854775808, 9223372036854775807], [-9223372036854775808, 9223372036854775807], [-9223372036854775808, 9223372036854775807], [-9223372036854775808, 9223372036854775807], [-9223372036854775808, 9223372036854775807]]
+[[-9223372036854775807, 9223372036854775808], [-9223372036854775807, 9223372036854775808], [-9223372036854775807, 9223372036854775808], [-9223372036854775807, 9223372036854775808], [-9223372036854775807, 9223372036854775808], [-9223372036854775807, 9223372036854775808], [-9223372036854775807, 9223372036854775808], [-9223372036854775807, 9223372036854775808], [-9223372036854775807, 9223372036854775808], [-9223372036854775807, 9223372036854775808]]
```

Comment:
how about add a array like `[[], null, [null], 1]` which contains null, normal data and zero value